src/h3lib/lib/edge.c

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright 2022 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file  edge.c
+ * @brief Edge functions for manipulating (nondirected) edge indexes.
+ */
+
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "algos.h"
+#include "constants.h"
+#include "coordijk.h"
+#include "h3Index.h"
+#include "latLng.h"
+#include "vertex.h"
+
+/**
+ * Wrap the error code from a directed edge function and present
+ * undirected edge errors instead.
+ */
+H3Error wrapDirectedEdgeError(H3Error err) {
```

Comment:
Nit: Assuming this is only used here, should it be `static`? We've gone back and forth on underscore prefix and `static` for single-file functions, but I thought the general decision was `static` and no underscore, right?

Same comment for the other non-exported functions below.